lib/reports/cost_report.rb

```diff
@@ -60,7 +60,11 @@ def pd_cost
     end
 
     def pd_cost_for_member(member)
-      (pd_cost / total_weight) * active_members[member.to_s].weight
```

Comment:
"hathitrust" already has a weight of 0.0 in `002._data.sql` ?

Reply:
Ah good catch, this was something I added while everything was broken and didn't remove.
Removing.